### PR TITLE
cleanup(compose): get rid of xjoin by upgrading inventory

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by skateman** [comment]
@LightOfHeaven1994 seems like eph is still down? @romanblanco tested this, now it works well, but you'd need to drop the old inventory image after merge as it no longer uses xjoin (faster)

**Review by vkrizan — APPROVED** [review]
You have my blessing.

**Pull request merged**
